src/3DMath.cpp

```diff
@@ -88,16 +88,16 @@ void Normalize(float v[3])
 #endif // WIN32_ASM
 }
 
-void InverseTransformVectorNormalize2x(float src0[3], float src1[3], float dst0[3], float dst1[3], float mtx[4][4] )
+void InverseTransformVectorNormalize2x(float src[2][3], float dst[2][3], float mtx[4][4] )
 {
-	InverseTransformVectorNormalize(src0, dst0, mtx);
-	InverseTransformVectorNormalize(src1, dst1, mtx);
+	InverseTransformVectorNormalize(&src[0], &dst[0], mtx);
```

Comment:
Visual says:
3dmath.cpp(93): error C2664: 'void InverseTransformVectorNormalize(float [],float [],float [][4])': cannot convert argument 1 from 'float (*)[3]' to 'float []'
3>        gliden64\src\3dmath.cpp(93): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast